new-frontiers-14/frontier-station-14
# Adds colored light bulbs

## About the PR



This PR adds new colored light bulbs based on the color types available in crystals. In addition to a mappable and discoverable box of bulbs. 

Furthermore it adds construction graphs to allow the bulbs to be made in the same fashion as LED tubes.

It now also offers default fills for all lightbulbs, in addition to a two new lightbulb types to be better aligned with some tube types. These new lightbulbs have lathe recipes.

## Why / Balance


When I was making the Nightlight shuttle I was dismayed at the lack of lighting options available when making "after hours" mood lighting for the bar. Adding colored bulbs, which can be used in additional devices, served as a great solution for this.

In short: More flexibility, more options.

## How to test


Test bulbs in sockets. 
Attempt to craft all bulbs.
Check box for proper bulb contents.
Double check lathe recipes.
Double check lightbulb illumination values to ensure that they're more or less where they should be. 

## Media

![ss+(2024-12-30+at+11 05 14)](https://github.com/user-attachments/assets/5a9ba88b-d250-4bdf-91e4-dff248274ff6)
![ss+(2024-12-26+at+03 41 34)](https://github.com/user-attachments/assets/426d612a-5dd5-4d39-8e3e-e3e269a91858)
![ss+(2024-12-31+at+11 01 30)](https://github.com/user-attachments/assets/da9dfccb-2046-4ea7-bf13-c5acc27c8b49)

## Requirements



## Breaking changes


**Changelog**



🆑 
- add: Colored craftable lightbulbs and prefilled light strobes.!
